Revise "Web libraries and packages" page (#5621)
dart-lang/site-www: src/content/web/libraries.md

description: Libraries and packages that can help you write Dart web apps.
---

Dart provides several packages and libraries to support

> web app development, the recommended option being [package:web][].

Reviewer: Perhaps it's only me, but if using the `package:<name>` or `dart:<name>` syntax, I personally prefer to put it in code syntax. What do you think (here and elsewhere on the page)?

Author: Definitely not just you! The original page didn't code syntax any of them, so that tripped me up. Thanks for catching that!

The [Dart SDK][] also contains other libraries that provide low-level web APIs.

## Web solutions

[Migrate to package:web][migrate]
: Learn how to migrate to `package:web`
  from Dart's previous web solution, [dart:html][].

[package:web API reference][package:web]

> : Complete reference documentation for `package:web` on pub.dev.

Reviewer 2: I wonder if it might be better to give a one-line summary of what this package is, instead of indicating that this link goes to it's documentation. Doing that for this and `js_interop` below might help a reader more quickly identify the right link for them without having to click on them all.

Author: That's a great idea, added!

[JavaScript interoperability documentation](/interop/js-interop)
: Interact with existing JavaScript or TypeScript libraries
  using Dart's JS interop support.

[The dart:js_interop documentation][js_interop]
: Complete reference documentation for the `dart:js_interop` library.

[Flutter web support][flutter-web]
: The [Flutter framework][flutter] supports web development with Dart,
  in addition to mobile, desktop, and embedded device support.

[Build a web app with Dart](/web/get-started)
: A quick overview of how to build, run, and debug a web app with Dart.

> To find more support for writing web apps, 

Reviewer: "To find more support" reads to me as the link will lead to me docs not packages, so I think the previous text or some other modification is likely a bit more clear.

Perhaps:

```suggestion
To find other libraries that support the web platform,
```
